_posts/2023-03-08-february-rubygems-updates.md

```diff
@@ -19,28 +19,32 @@ The following improvements and fixes are included in these releases (see the cha
+
+- added an experimental feature for `gem exec` command to run executables from gems that may or may not be installed - [#6309](https://github.com/rubygems/rubygems/pull/6309).
 - safe load all marshaled data - [#6384](https://github.com/rubygems/rubygems/pull/6384).
+- Make gemspec file generated by bundle gem properly exclude itself from packaged gem - [#6339](https://github.com/rubygems/rubygems/pull/6339).
+- preserved `bundler-setup-relative` paths if the `:path` option is set to relative in standalone setup - [#6327](https://github.com/rubygems/rubygems/pull/6327).
 
 In February, RubyGems gained [114 new commits](https://github.com/rubygems/rubygems/compare/master@%7B2023-02-01%7D...master@%7B2023-02-28%7D), contributed by 16 authors. There were 1,767 additions and 234 deletions across 106 files.
 
 ## RubyGems.org News
 
-This month, we made a lot of progress working on the backend admin dashboard. We implemented robust auditing of all changes, and added support for resetting users MFA, blocking a user, and deleting webhooks. 
+This month, we made significant progress on the backend admin dashboard. We implemented robust auditing of all changes and added support for resetting user's MFA, blocking a user, and deleting webhooks. 
 
-The dependency API deprecation has been [announced](https://blog.rubygems.org/2023/02/22/dependency-api-deprecation.html), and we have a plan for brownouts & turning off the endpoint entirely.
+We have [announced](https://blog.rubygems.org/2023/02/22/dependency-api-deprecation.html) the deprecation of the dependency API, and we plan to implement brownouts and remove the endpoint entirely. We migrated all RDS instances to Terraform and tested the migration of managed node groups.
 
 In addition to these updates, RubyGems.org saw several bug fixes and updates, some of which include:
 
-- set up datadog for application performance monitoring
 - added telemetry to capture MFA login durations - [#3376](https://github.com/rubygems/rubygems.org/pull/3376).
+- integrated DataDog for application performance monitoring - [#3461](https://github.com/rubygems/rubygems.org/pull/3461)
 - set up GitHub OAuth to protect the new /admin namespace - [#3388](https://github.com/rubygems/rubygems.org/pull/3388).
 - updated Rails test job name to be stable across version updates - [#3420](https://github.com/rubygems/rubygems.org/pull/3420).
 - fixed test avo warnings by removing redundant rake tasks loading - [#3422](https://github.com/rubygems/rubygems.org/pull/3422).
 - added avo MFA reset admin action & view of audit entries - [#3426](https://github.com/rubygems/rubygems.org/pull/3426).
 - updated `erd.dot` to fix ERD CI - [#3490](https://github.com/rubygems/rubygems.org/pull/3490).
+- updated the following packages - terraform `0.13.7 -> 1.3.9`, AWS terraform `2.51 -> 4.54`, external `1.2 -> 2.2`, kubernetes `1.8 -> 2.18`, and template `2.1 -> 2.3`.
```

Comment:
There was terraform upgraded and also individual terraform providers. I think it is ok to make this into 2 individual points.

```
- updated terraform `0.13.7 -> 1.3.9`
- updated terraform providers
  - AWS `2.51 -> 4.54`
  - external `1.2 -> 2.2`
  - kubernetes `1.8 -> 2.18`
  - template `2.1 -> 2.3`.
```